android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -2158,13 +2155,8 @@ public void setDrmLicenseHeader(String[] header){
 
     public void setAnalyticsMeta(ReadableMap analyticsData) {
         this.analyticsMeta = analyticsData;
-        if (player != null && analyticsData != null && youboraPlugin == null && contentId != analyticsData.getString("contentId")) {
-            initialiseYoubora();
-        }
-        if (player != null && analyticsData != null && youboraPlugin != null && youboraPlugin.getAdapter() != null && contentId != analyticsData.getString("contentId")) {
-            youboraPlugin.getAdapter().unregisterListeners();
-            youboraPlugin.getAdapter().fireStop();
-            youboraPlugin = null;
+
+        if (player != null && youboraPlugin == null && (analyticsMeta != null && analyticsMeta.getBoolean("contentIsLive") && contentId != analyticsMeta.getString("contentId"))) {
```

Comment:
reason we need `contentIsLive` check is because for VOD the setAnalyticsMeta gets called after player is initialised so we cant initialise youbora inside `finishPlayerInitialization` method coz analyticsMeta will be null